css/src/components/tabs.scss

```diff
@@ -0,0 +1,110 @@
+$tabs-padding: 0.5em 1.25em !default;
+$tabs-padding-large: 1em 1.5em !default;
+
+$tab-font-color: $primary !default;
+$tabs-hover-background-color: $primary-background !default;
+$tabs-hover-border-color: $primary-hover !default;
+$tabs-active-hover-background-color: $primary-hover !default;
+$tabs-active-hover-border-color: $primary-hover !default;
+$tabs-active-background-color: $primary-active;
+$tabs-active-border-color: $primary-active !default;
+$tabs-active-color: $text-invert !default;
+
+.tabs {
+	display: grid;
+	justify-content: center;
+	width: 100%;
+	grid-gap: 1em;
+	grid-template-columns: max-content auto max-content;
+	white-space: nowrap;
+
+	.tab-list {
```

Comment:
Namespace classnames with their parents: `tabs` > `tabs-list` (not tab-list)

Reply:
ditto for other classes

Reply:
updated